docs/computer_setup.md

```diff
@@ -2,82 +2,161 @@
+	with computers, but they do not work very well when we want to start automatting
+	complex tasks. If you want to do the same (complicated) thing to dozens or hundreds
+	of files it becomes slow, tedious and error prone to do that using a GUI. For those
+	kinds of tasks, a command line interface is much more appropriate. Command line
+	interfaces are interacted with exclusively via a keyboard (they were invented
+	before mice existed!):
 	
-	Basic command lines to interact with bash:
-	* ls: list directory contents. The ls command allows you to quickly view all files within a specified directory.
-	* pwd: print working directory you are in. If you want to know the exact directory you are working within, then pwd will tell you. 
-	* cd: change directory you are in.
-	* exit: exit out of the directory. This command will close a terminal window, and end the execution of a shell script. 
+	- Here are some commonly used commands in the Bash terminal:
+	- `ls`: list directory contents. The `ls` command allows you to quickly view all
+	files within your current working directory (just run `ls`) or any specified
+	directory (run `ls <path_name>` by replacing `<path_name>` with the path you are
+	interested in).
+	- `pwd`: print working directory. This prints the full path to the current
+	directory you are working in. 
+	- `cd`: change directory. If you run this by itself, it will move you to your "home"
+	directory. If you pass either a relative or full path to it (`cd <path_name>`) you
+	will move to that path. You can confirm that you moved by running `pwd` and you can
+	see what is inside the folder you are currently in by running `ls`.
 
-* Git
-	Git is a version control system that lets you track who made changes to what and when on github.com. It also makes collaboration easier, allowing changes by multiple people to all be merged into one source. You can access the git via command line (terminal) or a desktop app that has a GUI, such as GitKraken. 
-
-* GitHub
-	* Github is a software that runs locally on your computer and your files and their history are stored on your computer. Having a centrally located place to upload files and make changes and download changes from others enables you to collaborate more easily with other developers. 
+- Git
+	Git is a version control system that lets you track who made changes to what files
+	and at what time. A folder of code that is tracked together by git is called a
+	repository. Git runs locally on your computer and keeps track of the changes you
+	make and allows you to see all the past changes you and others have made. It also
+	makes collaboration easier by making it easy to coordinate changes by multiple
+	people. You can access the git via command line (terminal) or a desktop app that
+	has a GUI, such as GitKraken (strongly recommended!). 
 
+- GitHub
+	- Github is a website that provides hosting of git repositories and adds some very
```

Comment:
Consider: "provides hosting of git repositories, the way that Google Drive provides hosting of spreadsheets and slide decks " ?